.ci/buildoperations.py

```diff
@@ -33,22 +33,22 @@ def get_environment(cxxid: str, platform: str):
 
     if platform == "arm":
         if cxxid == "gcc":
-            env["CC"] = get_full_path("arm-none-eabi-gcc")
-            env["CXX"] = get_full_path("arm-none-eabi-g++")
+            env["CC"] = get_full_path("/opt/arm-gnu-toolchain/bin/arm-none-eabi-gcc")
```

Comment:
This I don't quite understand:
Shouldn't the build work without this script as well? Is this not purely a cmake thing?